src/repo.rs

```diff
@@ -59,6 +59,24 @@ impl Repository {
         }
     }
 
+    /// Find and open a repository, with additional options.
```

Comment:
Sure, I can add more documentation here.

(It's unfortunate that I can't just copy the documentation from the comments in libgit2, because the license isn't the same.)